corehq/apps/app_manager/management/commands/flag_domains_using_search_filter.py

```diff
@@ -0,0 +1,37 @@
+from django.core.management import BaseCommand
+
+from corehq import toggles
+from corehq.apps.app_manager.dbaccessors import domain_has_apps, get_app_ids_in_domain, get_current_app
+from corehq.util.log import with_progress_bar
+
+
+class Command(BaseCommand):
+    help = "One-time command to enable search filter flag where 'search filter' is already in use"
+
+    def handle(self, **options):
+        flag_domains_using_search_filter()
+
+
+def flag_domains_using_search_filter():
+    domains = (
+        set(toggles.USH_CASE_CLAIM_UPDATES.get_enabled_domains())
+        - set(toggles.USH_SEARCH_FILTER.get_enabled_domains())
+    )
+    for domain in with_progress_bar(domains):
```

Comment:
(also discussed earlier) If you wanted to work with chunking here that would be a good thing to get familiar with, although the estimated volume of ~300 domains doesn't demand it.